src/amo/languages.js

```diff
@@ -545,6 +545,10 @@ export const unfilteredLanguages = {
     English: 'Swahili',
     native: 'Kiswahili',
   },
+  szl: {
```

Comment:
is this new locale supported in Pontoon?

Reply:
Can I ask why the question? But yes, they're [enabled in Pontoon](https://pontoon.mozilla.org/szl/), and have been working on Firefox for several months.

Note that [we need](https://mozilla-l10n.github.io/documentation/products/firefox_desktop/adding_release.html#update-language-pack-settings-on-amo) the language listed here in other to make Firefox language packs work as expected on AMO, unless something has changed in the meantime.

Reply:
> Can I ask why the question? But yes, they're [enabled in Pontoon](https://pontoon.mozilla.org/szl/), and have been working on Firefox for several months.

I was wondering if we could add support for this new language in AMO itself, which I think will be possible.

> Note that [we need](https://mozilla-l10n.github.io/documentation/products/firefox_desktop/adding_release.html#update-language-pack-settings-on-amo) the language listed here in other to make Firefox language packs work as expected on AMO, unless something has changed in the meantime.

Yes, we ignore langpacks that are not in this list for some reasons.

Reply:
> I was wondering if we could add support for this new language in AMO itself, which I think will be possible.

Ah, makes sense. Unfortunately AMO is a very large project, and often these new locales are very small, so they tend to focus on products.